generators/2.0/java/templates/spring/src/main/java/GifFormActivity.java.ejs

```diff
@@ -0,0 +1,49 @@
+package <%= basePackage %>;
+
+import com.symphony.bdk.core.activity.ActivityMatcher;
+import com.symphony.bdk.core.activity.command.CommandContext;
+import com.symphony.bdk.core.activity.form.FormReplyActivity;
+import com.symphony.bdk.core.activity.form.FormReplyContext;
+import com.symphony.bdk.core.activity.model.ActivityInfo;
+import com.symphony.bdk.core.activity.model.ActivityType;
+import com.symphony.bdk.core.service.MessageService;
+import com.symphony.bdk.spring.annotation.Slash;
+import com.symphony.bdk.template.api.TemplateException;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import static java.util.Collections.emptyMap;
+
+@Slf4j
+@Component
+public class GifFormActivity extends FormReplyActivity<%-'<'%>FormReplyContext<%-'>'%> {
```

Comment:
Could we propose another example?

Reply:
What do you mean another example? Just change the `/gif` to something else (the name of the activity as well) or using `CommandActivity` instead of`FormActivity`?